docs/plugin-writer/tasks/add-remove.rst

```diff
@@ -1,8 +1,27 @@
 Adding and Removing Content
 ===========================
 
-TODO: RepositoryVersion
-https://pulp.plan.io/issues/3220
+Repositories have versions. A new respository version is created when its set of content units
+changes.
+
+To facilitate the creation of repository versions a :class:`~pulpcore.plugin.models.RepositoryVersion`
+context manager is provided. Plugin Writers are strongly encouraged to use RepositoryVersion as
+a context manager to provide transactional safety, working directory set up, and cleaning
+up the database on failures::
+
+     with RepositoryVersion.create(repository) as new_version:
+
+        # add content manually
+        new_version.add_content(content)
```

Comment:
could we add these options (commented out, if they won't work) to the template code block?